nncf/torch/pruning/filter_pruning/algo.py

```diff
@@ -110,8 +113,8 @@ def _is_pruned_module(self, module) -> bool:
         return isinstance(module, tuple(NNCF_PRUNING_MODULES_DICT.keys()))
 
     def get_op_types_of_pruned_modules(self) -> List[str]:
-        types = [v.op_func_name for v in NNCF_PRUNING_MODULES_DICT]
-        return types
+        return [op_name for meta_op in [PTConvolutionPruningOp, PTTransposeConvolutionPruningOp, PTLinearPruningOp]
+                for op_name in meta_op.get_all_op_aliases()]
```

Comment:
What is a motivation of this change? Why we are keeping `NNCF_PRUNING_MODULES_DICT` in `_is_pruned_module` method?

Reply:
It will be more aligned with TF. 
I'd remove NNCF_PRUNING_MODULES_DICT if it can be implemented via PruningOp's like in TF part.

Reply:
Originally, I thought `NNCF_PRUNING_MODULES_DICT` is not enough, because it contains just `NNCFLinear` module and doesn't contain other subtypes (e.g. [addmm](https://pytorch.org/docs/stable/generated/torch.matmul.html), [matmul](https://pytorch.org/docs/stable/generated/torch.addmm.html)).
But it seems, we support only pruning of linear modules in PyTorch, not weight-less operations that have 2 input tensors. Therefore, it makes sense to revert this change. Thanks for highlighting!